### PR TITLE
Fix autocompete bug preventing it from becoming active when value is added from code

### PR DESCRIPTION
**Pull request by lukelast**

### CODE REVIEW

File: src/main/java/gwt/material/design/addins/client/autocomplete/MaterialAutoComplete.java [hunk 1]
```diff
@@ -442,15 +442,14 @@ public void setItemValues(List<String> itemValues) {
...
         for (String value : itemValues) {
             Suggestion suggestion = new gwt.material.design.client.base.Suggestion(value, value);
             list.add(suggestion);
         }
-        if (itemValues.size() > 0) {
+        setValue(list);
+		if (itemValues.size() > 0) {
             lblPlaceholder.addStyleName(CssName.ACTIVE);
         }
-        setValue(list);
```
Comment (lukelast) [inline src/main/java/gwt/material/design/addins/client/autocomplete/MaterialAutoComplete.java]: This method will clear the ACTIVE style so the style must be added afterwards.

**Comment by lukelast** [comment]
Should I file an issue for this instead? Or did I pick the wrong branch?

**Comment by kevzlou7979** [comment]
Hi @lukelast  Thanks for this fix, kindly solve the conflict first then we can merge this in.

**Comment by lukelast** [comment]
whoops I didn't see that conflicts arose. Looks like your other change actually fixes the issue also, still some value in this PR I think though.
Thanks

**Comment by kevzlou7979** [comment]
Yeah I will merge this on soon, just solving some issues atm

**Pull request merged**